content/blog/2024-03-09-xmpp-providers-chat.md

```diff
@@ -0,0 +1,19 @@
+---
+title: XMPP Providers Chat
+date: 2024-03-09
+---
+
+## Come Join Us
+
+After [automating](/blog/2023-12-29-xmpp-providers-fully-automated/) almost all the work necessary for querying [provider properties](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/README.md#properties), we were able to start adding new providers! There are 59 providers on the list at this moment :tada: which enables us to show some [statistics](/statistics/) as well.
```

Comment:
Great proposal! Let's call to get the 200 complete in 2024?

Reply:
Nah, let's be humble ;)